plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerConnectorTest.java

```diff
@@ -170,6 +170,31 @@ public void testCreateAndDropTableWithSpecialCharacterName(String tableName)
         assertFalse(getQueryRunner().tableExists(getSession(), tableName));
     }
 
+    // TODO remove this test after refactoring testCreateAndDropTableWithSpecialCharacterName
+    @Override
+    @Test(dataProvider = "testTableNameDataProvider")
+    public void testRenameColumnName(String columnName)
+    {
+        super.testRenameColumnName(columnName);
+    }
+
+    // TODO move this test to BaseConnectorTest
+    @Test(dataProvider = "testTableNameDataProvider")
+    public void testRenameFromToTableWithSpecialCharacterName(String tableName)
+    {
+        // Until https://github.com/trinodb/trino/issues/17 the table name is effectively lowercase
+        tableName = tableName.toLowerCase(ENGLISH);
+        String tableNameInSql = "\"" + tableName.replace("\"", "\"\"") + "\"";
+        String sourceTableName = "test_rename_source_" + randomTableSuffix();
+        assertUpdate("CREATE TABLE " + sourceTableName + " AS SELECT 123 x", 1);
+
+        assertUpdate("ALTER TABLE " + sourceTableName + " RENAME TO " + tableNameInSql);
+        assertTrue(getQueryRunner().tableExists(getSession(), tableName));
```

Comment:
looks redundant if you also select from it

Reply:
removed